src/components/Tabs/Tabs.cy.tsx

```diff
@@ -53,13 +53,15 @@ const data: TabItemProps[] = [
     },
 ];
 
-const TabComponent = () => {
+const TABS_DATA_TEST_ID = '[data-test-id=tabs]';
+
+const TabComponent = ({ paddingX }: { paddingX?: TabsPaddingX }) => {
     const [activeItemId, setActiveItemId] = useState(data[0].id);
     return (
         <Tabs
             activeItemId={activeItemId}
             onChange={(value) => setActiveItemId(value)}
-            paddingX={TabsPaddingX.Small}
+            paddingX={paddingX}
```

Comment:
I think this change is not necessary as we are not changing the type and decided to keep `TabsPaddingX`